lib/services/components.js

```diff
@@ -140,16 +144,17 @@ function put(uri, data) {
 }
 
 /**
- * Clear all of an object properties (in place), not a new object
- *
- * No need to return anything since it's in-place
+ * Clear all of an object properties (in place), not a new object.
  *
  * @param {object} obj
+ * @returns {object}
  */
 function clearOwnProperties(obj) {
   _.forOwn(obj, function (value, key) {
     delete obj[key];
   });
+
+  return obj;
```

Comment:
For convenience mostly.  For example, in coffee-script, the result of the last line is returned always because of this reason.

It's a useful convention to not have functions that don't return anything so that statements can be combined.